src/app/containers/CpsOnwardJourney/index.jsx

```diff
@@ -127,6 +178,11 @@ const CpsOnwardJourney = ({
   sectionLabelBar,
   sectionLabelBackground,
   columnType,
+  hasSkipLnk,
+  endTextId,
+  skipText,
+  endTextVisuallyHidden,
+  skipLinkTerms,
```

Comment:
As all these props relate to the skip link, could we put them in a nested object to keep them isolated from the other props:
```
skipLinkConfig: {
  endTextId,
  skipText,
  endTextVisuallyHidden,
  skipLinkTerms,
}
```
I've removed `hasSkipLnk` (typo) as the presence of the skipLinkConfig indicates the onward journey has a skip link